docs/styleguide.md

```diff
@@ -226,9 +238,12 @@ The properties chapter must be in following form:
 ...
 ```
 
-The title can be `###` or `####`-levels depending on whether it is a property of
-a module or a class.
+The heading can be `###` or `####`-levels depending on whether the property
+belongs to a module or a class.
 
 ## Documentation Translations
```

Comment:
Should be `Document translations` per the title case documentation?

Reply:
True!